docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.    lvals = [(1.0 - opt_fval) / (num2 / cycle_step) 
</pre><p>Please add a small description of this formula or a reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Outdated, Loschmidt echo removed.</p>